assets/css/styles.css

```diff
@@ -0,0 +1,276 @@
+:root {
+  --very-dark-green: #1a3c40;
+  --dark-green: #1d5c63;
+  --green: #417d7a;
+  --beige: #ede6db;
+}
```

Comment:
Make sure you leave spare lines between the different CSS blocks to make it easier to read